.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
+/workspace.xml
```

Comment:
Isn't is simply easier to  ignore the whole .idea folder in git ignore or even better to ignore it on your system via a global gitignore file (that's what I do with my editor configs)

Reply:
Removed.
Thanks for your suggestion.